rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -772,6 +804,31 @@ void Z3ConvVisitor::VisitBinaryApp(z3::expr z_op) {
                                   ast_ctx->BoolTy);
       break;
 
+    case Z3_OP_SLEQ:
```

Comment:
See above comments on signedness. Correct behavior should probably be enforced via `clang::CStyleCastExpr` expressions. This goes for quotient (division), remainder and comparison operations alike.